RecoLocalTracker/SiPixelRecHits/python/PixelCPEFast_cfi.py

```diff
@@ -0,0 +1,38 @@
+import FWCore.ParameterSet.Config as cms
+
+PixelCPEFastESProducer = cms.ESProducer("PixelCPEFastESProducer",
```

Comment:
this should clone from the fill description `RecoLocalTracker.SiPixelRecHits.PixelCPEFastESProducer_cfi` with appropriate replace if any

Reply:
The content of this file is actually identical to that of the autogenerated cfi file.
I would simply delete this file, and use the autogen one instead.

Reply:
Done in https://github.com/cms-patatrack/cmssw/pull/593/commits/78337cc9a366536b81525795e6d378834cff44a7 .